src/mbgl/geometry/buffer.hpp

```diff
@@ -36,11 +37,11 @@ class Buffer : private util::noncopyable {
 
     // Transfers this buffer to the GPU and binds the buffer to the GL context.
     void bind(gl::ObjectStore& store) {
-        if (buffer.created()) {
-            MBGL_CHECK_ERROR(glBindBuffer(bufferType, getID()));
+        if (buffer) {
+            MBGL_CHECK_ERROR(glBindBuffer(bufferType, (*buffer).get()));
```

Comment:
Shouldn't `*buffer` work, using [the `unique_resource` implicit conversion](https://github.com/okdshin/unique_resource/blob/cba309e92ec79a95be2aa5a324a688a06af8d40a/unique_resource.hpp#L76)?